add "What Type Of Node You Need to Run For Archive Arbitrum One?" (#216)
OffchainLabs/arbitrum-docs: arbitrum-docs/node-running/running-an-archive-node.mdx

❗️Note: The minimum storage requirements will change over time as the Nitro chain grows. It is recommended to use more than the minimum requirements to run a robust archival node.

> ### What Tyoe Of Node You Need to Run For Archive Arbitrum One?

Reviewer: ```suggestion
### What Type of Node Do You Need to Run to Send Archive Requests on Arbitrum One?
```

> Because Arbitrum One genesis from Classic stack and running for 1 years (before block height 2207817), and now it upgrades to Nitro stack.

Reviewer 2: ```suggestion
Before the Nitro upgrade happened, Arbitrum One was running on Classic stack for about one year (before block height 2207817).
```

> Although we move Classic's latest state as Nitro's genesis state (see [state migration](../migration/state-migration.mdx)), but nitro stack can't serve arcive requests for pre-nitro blocks, so you have 3 options to run an Arbitrm One archive node:

Reviewer 2: ```suggestion
Although we have moved Classic's latest state as Nitro's genesis state (see [state migration](../migration/state-migration.mdx)), the Nitro stack can't serve archive requests for pre-nitro blocks. So you have three options to run an Arbitrum One in an archive mode:
```

> Although we have moved Classic's latest state as Nitro's genesis state (see [state migration](../migration/state-migration.mdx)), the Nitro stack can't serve archive requests for pre-nitro blocks. So you have three options to run an Arbitrum One in an archive mode:

Reviewer: ```suggestion
Although we have moved Classic's latest state as Nitro's genesis state (see [state migration](../migration/state-migration.mdx)), the Nitro stack can't serve archive requests for pre-nitro blocks. So you have three options to run Arbitrum One in archive mode:
```

> 1. Only post-nitro archive requests needs, you can see [running-a-node](./running-a-node.md).

Reviewer 2: ```suggestion
1. If you **only** need to send post-nitro archive requests, you can see [running-a-node](./running-a-node.md).
```

> 2. Only pre-nitro archive requests needs, you can see [running-a-classic-node](./running-a-classic-node.mdx).

Reviewer 2: ```suggestion
2. If you **only** need to send pre-nitro archive requests, you can see [running-a-classic-node](./running-a-classic-node.mdx).
```

> 3. Both post-nitro and pre-nitro archive requests needs, then you should run not only archive nitro node but also archive classic node, this docs will tell you how to do this.

Reviewer 2: ```suggestion
3. If you need **both** post-nitro and pre-nitro archive requests, you should run **both** an archive Nitro node **and** an archive classic node, there we  explain how to do this.
```

> 3. If you need **both** post-nitro and pre-nitro archive requests, you should run **both** an archive Nitro node **and** an archive classic node, there we  explain how to do this.

Reviewer: ```suggestion
3. If you need **both** post-nitro and pre-nitro archive requests, you should run **both** an archive Nitro node **and** an archive classic node, in the following we will explain how to do this.
```

> ### Required Artifacts

Reviewer: all headers from here should be one hierarchy below `### What Tyoe Of Node You Need to Run For Archive Arbitrum One?` to create the right content hierarchy

so either the previous two are changed to `##` or all of the following ones should be `####`

- You could run both Arbitrum One and classic in archive mode. Arbitrum One has been upgraded to Nitro, the latest Arbitrum tech stack; "Arbitrum Classic" is our term for the old, pre-Nitro tech stack. The Nitro node databases have the raw data of all blocks, including pre-Nitro blocks. However, Nitro nodes cannot execute anything on pre-Nitro blocks. Arbitrum Nova started as a Nitro chain, so has no classic blocks.
- Here we provide examples for running both Arbitrum One and Arbitrum Classic in archival mode.
  
- An example of how to run an Arbitrum Classic archival node:
  ```shell
  docker run --rm -it -v /some/local/dir/arbitrum-mainnet/:/home/user/.arbitrum/mainnet -p 0.0.0.0:8547:8547 -p 0.0.0.0:8548:8548 offchainlabs/arb-node:v1.4.5-e97c1a4 --l1.url=https://l1-node:8545/ --node.chain-id=42161 --l2.disable-upstream --node.cache.allow-slow-lookup --core.checkpoint-gas-frequency=156250000
  ```

> - An example of how to run an Arbitrum One archival node:

Reviewer 3: Should we change this from Arbitrum One to Arbitrum Nitro too?
Also, latest version is v2.0.13-174496c

Author: Yeah, I planed to do that when this pr get merged since other file also contain outdate image version.
